.github/workflows/dea-intertidal-image.yml

```diff
@@ -52,14 +52,27 @@ jobs:
           # Run integration tests using Docker
           docker compose run dea_intertidal pytest -v --cov=intertidal --cov-report=xml tests
 
+          # Copy out validation outputs produced by the integration tests and place them
```

Comment:
We can't commit directly inside the container, so the files need to first be copied from the mounted "artifacts" directory, then copied into their correct locations on the main Github runner (outside of the container). We can then commit them via Git in the next step